samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,393 @@
+
+You need the nRF Connect Device Manager app for SUIT update over Bluetooth Low Energy:
+
+* `nRF Device Manager mobile app for Android`_
+  (The minimum required version is v1.9.)
+
+* `nRF Device Manager mobile app for iOS`_
+  (The minimum required version is v1.5.)
+
+For a SUIT update over UART, you need to install MCUmgr.
+MCUmgr is a tool that can be used to upload SUIT envelopes through the SMP protocol.
+For more information on MCUmgr installation, see :ref:`zephyr:mcu_mgr`.
+
+Overview
+********
+
+The sample uses one of the following to perform an update on the nRF54H20 SoC, featuring Nordic Semiconductor’s implementation of the SUIT procedure:
+
+* Bluetooth Low Energy and the nRF Connect Device Manager app
+* UART and MCUmgr
+
+The sample demonstrates how to perform a DFU of the Application and Radio firmware.
+
+SUIT is the only DFU procedure supported for the nRF54H20 SoCs.
+To read more about the SUIT procedure, see :ref:`ug_nrf54h20_suit_intro`.
+
+|NCS|-specific SUIT limitations
+===============================
+
+The sample and the SUIT DFU procedure currently only support the Application Core and the Radio Core firmware updates.
+Additionally, this sample is an early version and is only meant to showcase the start of Nordic Semiconductor’s implementation of the SUIT DFU procedure.
+
+User interface
+**************
+
+LED 1:
+    The number of blinks indicates the application firmware version.
+    The default is set to ``1`` to blink once, indicating "Version 1".
+
+Configuration
+*************
+
+|config|
+
+The default configuration uses UART with Version 1.
+To change the version of the application, configure the :ref:`CONFIG_APP_VERSION <CONFIG_APP_VERSION>` Kconfig option.
+This also changes the number of blinks on **LED 1** and sets the :ref:`sequence number <ug_suit_dfu_suit_manifest_elements>` of the SUIT :ref:`envelope <ug_suit_dfu_suit_concepts>`’s manifest.
+
+To use this configuration, include the ``-DCONFIG_APP_VERSION=x`` option with the build command, where x is the version number.
+
+For example:
+
+.. code-block:: console
+
+   west build -p -b nrf54h20dk/nrf54h20/cpuapp -- -DCONFIG_APP_VERSION=2
+
+If you do not specify this configuration, the sample is built with Version 1.
+
+Configuration options
+=====================
+
+Check and configure the following configuration option for the sample:
+
+.. _CONFIG_APP_VERSION:
+
+CONFIG_APP_VERSION - Configuration for the application version
+   The sample configuration updates the version of the application.
+   The default value is ``1``.
+
+Modify partition sizes
+======================
+
+You can also modify the size and location of the partitions.
+This is done by modifying the values for the desired location and size of the partition in the devicetree :file:`.overlay` files.
+
+* To modify the Application Core’s partition size,  modify the values for ``slot0_partition`` defined in :file:`modules/ic-next/dts/arm/nordic_nrf_next/nrf54h20_cpuapp.dtsi`.
```

Comment:
The new DT source file is: https://github.com/nrfconnect/sdk-zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi